lib-utilities/config/constants.go

```diff
@@ -142,3 +142,9 @@ var (
 		"TLS_1.2": tls.VersionTLS12,
 	}
 )
+
+// AllowedMessageBusTypes is for checking for message types are allowed
+var AllowedMessageBusTypes = map[string]bool{
+	"kafka":       true,
+	"RedisStream": true,
+}
```

Comment:
cases should match across the options, keep it either small/caps/camelcase for both options

Reply:
updated